generators/server/index.js

```diff
@@ -402,11 +403,11 @@ module.exports = JhipsterServerGenerator.extend({
 
         let logMsg =
             'Run your Spring Boot application:' +
-            '\n ' + chalk.yellow.bold('./mvnw');
+            '\n ' + chalk.yellow.bold('.' + path.sep + 'mvnw');
```

Comment:
@kaidohallik so on windows this would be `.mvnw` and on *nix `./mvnw` can you check if `.mvnw` works in windows as I dont have a windows machine now to check